CONTRIBUTING.md

```diff
@@ -1,72 +1,46 @@
-issue locally. This includes:
+## Security vulnerability
 
-* The version of AtoM you are using
-* Basic information about your installation environment, including PHP, MySQL,
-  Elasticsearch, and operating system versions
-* Steps to reproduce the issue
-* The resulting error or vulnerability and the expected outcome
-* If there are any error logs related to the issue, please include the
-  relevant parts as well
+**If you are reporting a security vulnerability**, refer yourself to the
+[SECURITY.md](./SECURITY.md) document. 
 
-You can find useful tips on how to find this information in the Troubleshooting 
-page in our documentation:
+## Reporting general bugs
 
-* https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/
+If you have discovered an issue in AtoM, we welcome your reports.
 
-Note as well that, as a community-driven open source project, we depend on our
-community to be able to maintain and develop AtoM. We are committed to
-including as many bug fixes as we can in each new release, but a confirmed
-report is not enough alone to guarantee that a fix will be included in the
-next release. If your institution is interested in sponsoring a fix, feel free
-to contact Artefactual Systems for an estimate - all sponsored development
-will be included in the next public release. For more information on how we
-develop and maintain AtoM, please see:
+Start by doing a search on [AtoM user forum](https://groups.google.com/forum/#!forum/ica-atom-users)
+and in the [AtoM GitHub issues](https://github.com/artefactual/atom/issues)
+to see if your question has been considered in the past.
 
-* https://wiki.accesstomemory.org/Development/Philosophy
+If nothing comes up, start a conversation by opening a new thread on the user
+forum. When your bug will have been better understood, you may be invited to
+open an issue on GitHub.
 
-We also welcome pull requests to fix issues!
+The [Troubleshooting guide](https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/troubleshooting/) contains the information needed to open an issue.
 
 ## Contributing code
 
-If you're considering contributing code to the project, please read our
-contribution page and familiarize yourself with our coding style and code
-review guidelines.
+Please read our contribution page and familiarize yourself with our coding
+style and code review guidelines.
 
-* Find out more about our Code Review process here: [Code review](https://wiki.accesstomemory.org/Development/Code_review)
-* Learn about our Coding standards here: [Coding standard](https://wiki.accesstomemory.org/Development/Coding_standard)
-* Find information about our Code repository here: [Code repository](https://wiki.accesstomemory.org/Resources/Code_repository)
-* Read about our Testing strategies here: [Testing](https://wiki.accesstomemory.org/Development/Testing)
+* Find out more about our [Code review](https://wiki.accesstomemory.org/Development/Code_review)
```

Comment:
You will probably want to review all these wiki pages with the Maintainers /  AtoM hosting team if you haven't already, as they may include recommendations that no longer match our current approach, FYI

Reply:
Thanks! Where would it be a good place to file issues/suggestions to update and follow wiki work in progress and ideas ?

Reply:
following our 2.9 release, we will focus on updating our documentation to reflect Solr as our new search engine. if you notice anything's missing in our documentations, we can address those changes then. feel free to create a list of changes or pull requests in atom docs repository for us to follow up on!

Reply:
If needed, we can open issues on GitHub about the wiki content - not optimal, but this will do for now.